core/src/main/java/bisq/core/trade/HavenoUtils.java

```diff
@@ -102,14 +106,18 @@ public static double atomicUnitsToXmr(BigInteger atomicUnits) {
         return new BigDecimal(atomicUnits).divide(new BigDecimal(XMR_AU_MULTIPLIER)).doubleValue();
     }
 
+    public static String formatToXmr(Coin xmrAmount) {
```

Comment:
Please move to after the conversion utils so the conversion utils remain grouped. After `xmrToCentineros()`.

Reply:
This issue remains.